src/anjaliS/PrimeNumbersInRange.java

```diff
@@ -0,0 +1,29 @@
+package anjaliS;
+
+import java.util.Scanner;
+
+public class PrimeNumbersInRange {
+	void displayPrime(int start, int end) {
+		for (int index = start; index <= end; index++) {
+			boolean flag = false;
+			for (int j = 2; j <= index / 2; j++) {
+				if (index % j == 0)
+					flag = true;
+			}
```

Comment:
Here you should use break, because once you know agr if condition get satisfies then that number is not prime number.
then no need to check further iterations